scripts/importing-Exporting-Config.js

```diff
@@ -0,0 +1,508 @@
+
+        for (var j = 0; j < event_Specific_Teams_Keys.length; j++) //loop through teams
+        {
+            var currTeam = event_Specific_Teams_Keys[j];
+            var currTeamName = getTeams(season,currEvent)[currTeam];
+            //alert(currTeam);
+            
+            var team_Obj = {};
+            var team_Obj_Name = `Data for team ${currTeam}:${currTeamName}`;
+            var team_Obj_Value = []; //will be of length 2: having a match obj and a pit obj
+            
+            var match_Obj = {};
+            var match_Obj_Name = `Match Data for team ${currTeam}`;
+            var match_Obj_Value = []; //length is unknown (dont know how many matches per event the team will attend)
+            
+            var pit_Obj = {}; 
+            var pit_Obj_Name = `Pit Data for team ${currTeam}`;
+            var pit_Obj_Value = ''; //not an array, only 1 value (only 1 pit entry per event)
+
+            var loc_Strg_Check = `/${season}/${currEvent}/${currTeam}/`; //to check if a key contains the curr event/curr team
+
+            for (var k = 0; k < loc_Strge_Keys.length; k++)
+            {
+                var curr_loc_Strg_key = loc_Strge_Keys[k];
+                if (!curr_loc_Strg_key.startsWith(loc_Strg_Check)) continue; //we know curr key has data on selected team for curr event
+
+                if (curr_loc_Strg_key.includes('Match Data')) //add to match_Obj
+                {
+                    var match_Data_Arr = JSON.parse(localStorage[curr_loc_Strg_key]); //is an array of smaller element/frequency object pairs
+                    match_Obj_Value.push(match_Data_Arr);
+                }
+
+                if (curr_loc_Strg_key.includes('pit')) //add to pit_Obj
+                {
+                    var pit_Data_Arr = JSON.parse(localStorage[curr_loc_Strg_key]); //is an array of smaller Q/A object pairs
+                    pit_Obj_Value = pit_Data_Arr;
+                }
+            }
+
+            //set up match/pit objects
+            match_Obj[match_Obj_Name] = match_Obj_Value;
+            pit_Obj[pit_Obj_Name] = pit_Obj_Value;
+
+            //add match/pit objects to team object
+            team_Obj_Value.push(match_Obj);
+            team_Obj_Value.push(pit_Obj);
+
+            //add team object to event object
+            team_Obj[team_Obj_Name] = team_Obj_Value;
+            event_Obj_Value.push(team_Obj);
+        }
+
+        //push event obj to total output array
+        event_Obj[event_Obj_Name] = event_Obj_Value;
+        total_Output_Arr.push(event_Obj);
+    }
+
+    return total_Output_Arr;
+}
+
+
+
+
+// Importing Season config \/\/\/
+function readImportFile()
+{
+    var file = $('#importFile').prop('files')[0];
+
+    var fr = new FileReader();
+    fr.onload = function(e) {
+        var content = e.target.result;
+        var obj = JSON.parse(content);
+
+        if (file.name.endsWith('.rcubedscoutconfig'))
```

Comment:
And what do you do if it does not match this condition?

Consider flipping these `if`s around:
```js
if(!file.name.endsWith(config_file_name_extension)) {
  return; // or whatever error handling you want
}

var continueOrNot = getImportModalChoice();
...
```